scripts/copy-apikey.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "Copying IBM Cloud apikey into development environment..."
+docker cp ~/.bluemix/apikey.json nyu:/home/devops 
```

Comment:
```suggestion
docker cp ../.vscode/apikey.json nyu:/home/devops 
```
Because you put the `apikey.json` in the folder `.vscode` and we need to share the same key, I think it should be changed.